tmt/steps/execute/__init__.py

```diff
@@ -387,6 +390,23 @@ def check_result_file(self, test):
             data['note'] = f"invalid test result '{result}' in result file"
         return tmt.Result(data, name=test.name, interpret=test.result)
 
+    def check_abort_file(self, test):
+        """
+        Check for an abort file created by tmt-abort
+
+        If it exists aise the AbortTestError
```

Comment:
I think currently it works as I'd expect it to. Abort stops the given test and also prevents other tests in the plan from running. I don't think it makes sense to prevent plans from running after abort, however that's just my opinion. However, I'd love to hear other's opinion on this

Reply:
Ah, now I understand. I'd say abort in one plan should not affect other plans now 
Maybe as a opt-in/out feature (could be useful for cascade of testing if we ever to that) to be added later.

Reply:
Okay. I've added some tests and left the functionality as is for the moment, such that if a test in a .sh file is aborted then subsequent tests in that particular file will not be executed, but tests from other files will be.